cloud-config-cluster.yml

```diff
@@ -1,3 +1,4 @@
+#cloud-config
```

Comment:
This is needed but was missing.  We don't really use this file any more.  It
s use case is building the old style cluster with es master on the front end machine.  Do we want to continue maintaining this deployment?